app/assets/javascripts/modules/map/spatial_selection.js.coffee

```diff
@@ -148,6 +150,8 @@ ns.SpatialSelection = do (window,
         @_layer = @_oldLayer
         @_oldLayer = null
         @_drawnItems.addLayer(@_layer)
+        currentPage.query.spatialCondition.visible(true)
```

Comment:
"visible" is a presentation concern, but the spatial condition is part of a data model, not a UI model. The condition shouldn't care whether or not its UI is visible; that should be tracked elsewhere.